data/base/script/campaign/libcampaign_includes/debug.js

```diff
@@ -140,7 +140,7 @@ function __camUpdateMarkedTiles()
 	hackMarkTiles();
 	if (camIsCheating() && camDef(__camMarkedTiles))
```

Comment:
`__camMarkedTiles` should be always an object. so it should be possible to remove the `&& camDef(__camMarkedTiles)`